app/extensions/string_extensions.rb

```diff
@@ -365,7 +365,7 @@ class String
       "nbsp"  => " "
     }.freeze
   end
-  # rubocop:enable Layout/AlignHash
+  # rubocop:enable Layout/HashAlignment,Layout/AlignHash
```

Comment:
```ruby
  # rubocop:enable Layout/HashAlignment
```
The cop name was change in v.77.
See https://github.com/rubocop-hq/rubocop/issues/7077#issuecomment-546317645